Library/ENV/scm/git

```diff
@@ -23,6 +23,10 @@ def exec(*args)
   # prevent fork-bombs
   arg0 = args.first
   return if arg0 =~ /^#{F}/i || Pathname.new(arg0).realpath == SELF_REAL
+  if ARGV == %w[--homebrew=print-path]
```

Comment:
If you don't feel too strongly about this, I'd prefer to keep it as-is. My mental model behind this is that `git --homebrew=<feature>` triggers Homebrew-internal behavior and `<feature>` happens to be `print-path` in this instance.

While testing and debugging, I actually had a second one called `list-paths` that would output all considered paths in lookup order. (But that's not something that belongs in this PR.)